universal_login/deploy.ts

```diff
@@ -59,7 +59,7 @@ const doDeploy = async (env: Env) => {
 // 2 req/second - this ensures we never hit that
 // https://auth0.com/docs/troubleshoot/customer-support/operational-policies/rate-limit-policy/management-api-endpoint-rate-limits
 const rateLimiter = new RateLimiter({
-  tokensPerInterval: 2,
+  tokensPerInterval: 1,
```

Comment:
Let's update the comment to explain that we've deliberately lowered it from the Auth0 threshold, so a well-meaning person doesn't come back to "fix" this later.